core/iwasm/common/wasm_runtime_common.c

```diff
@@ -3694,6 +4022,11 @@ wasm_externref_obj2ref(WASMModuleInstanceCommon *module_inst, void *extern_obj,
     ExternRefMapNode *node;
     uint32 externref_idx;
 
+    if (NULL_REF == (uint32)(uintptr_t)extern_obj) {
```

Comment:
Should not use NULL_REF, and should not used uint32, in 64-bit, uintptr_t is truncated.
Had better use `if (UINTPTR_MAX == (uintptr_t)extern_obj)`

Reply:
The code is trying to catch a variable with the initialized value with `ref.null`. So, it is a *NULL_REF* originally.